charts/opentelemetry-collector/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 nameOverride: ""
 fullnameOverride: ""
-
+commonLabels: {}
```

Comment:
Please add a description the explains this field similar to https://github.com/prometheus-community/helm-charts/blob/8b91a619e9407c0e830df7071ee73cd160937b6b/charts/kube-prometheus-stack/values.yaml#L25-L27.